src/Tools/Github/GitMergeBot/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "LibGit2Sharp": "0.22.0",
-    "LibGit2Sharp.NativeBinaries": "1.0.160",
+    "LibGit2Sharp.NativeBinaries": "1.0.129",
```

Comment:
I can do that here. We will still need the reference for the insertion tool, since the props aren't transitive and the binaries need to be linked to the exe output.